### ISSUE TITLE
Wrong stats being recorded & displayed

### ISSUE THREAD

**Issue by YvesW** [Epix37/Hearthstone-Deck-Tracker#743]
I'm only recording my stats in ranked games, since I concede a lot in casual games if I feel like it. Because of that reason, I want the tracker to display my competetive (ranked) stats. It displays all my stats though, while the casual stats shouldn't even be recorded.

Settings
![Settings](https://cloud.githubusercontent.com/assets/7929021/6474633/a72d828a-c202-11e4-8b7e-ab8c210548fd.PNG)

My ranked stats for a druid deck
![My ranked stats for a druid deck](https://cloud.githubusercontent.com/assets/7929021/6474654/eccebcdc-c202-11e4-8a22-9756a73b96b5.PNG)

All stats for the same druid deck (most of which shouldn't even get recorded)
![All stats for the same druid deck (most of which shouldn't even get recorded)](https://cloud.githubusercontent.com/assets/7929021/6474661/0c277f1a-c203-11e4-9cb7-dd83569b8a82.PNG)

Druid deck in overview mode (I want my ranked stats displayed here)
![Druid deck in overview mode (I want my ranked stats displayed here)](https://cloud.githubusercontent.com/assets/7929021/6474675/32302036-c203-11e4-8232-8bb2ce3e52df.PNG)

Old stats are fine (haven't played this deck in a few weeks)
![Old stats are fine (haven't played this deck in a few weeks)](https://cloud.githubusercontent.com/assets/7929021/6474736/bbf3e35c-c203-11e4-9037-d9905e44c3a5.PNG)

**Comment by Epix37** [comment]
So, one non-ranked game got recorded on the druid deck, yes?
And the game that's not supposed to be there is a casual one?

Is this consistent? (Do all e.g. casual games get recorded even though they're not supposed to?)

**Comment by YvesW** [comment]
Yes, one non-ranked game got recorded indeed. The game that shouldn't be there is a casual one indeed.
I went into casual with the Druid deck, played 3 turns and conceded (the tracker told me I'd also conceded the hunter game). It didn't register.

Thanks for the tip by the way; I found the game. Its mode was 'none'. The other games' modes were all 'ranked' except for the last ranked game that I played yesterday, which also had the mode 'none' (doesn't show my rank as well). Not sure if this is important for you.
